spec/rails_helper.rb

```diff
@@ -71,4 +71,14 @@
   config.filter_rails_from_backtrace!
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
+  DatabaseCleaner.strategy = :truncation
```

Comment:
Should I remove this? Database cleaner wasn't working for me before adding these changes